src/checks.cc

```diff
@@ -116,7 +116,8 @@ LinterResult CheckUppercaseKeywords(absl::string_view sql,
       if (option.IsActive(ErrorCode::kLetterCase, position))
         result.Add(
             ErrorCode::kLetterCase, sql, position,
-            absl::StrCat("All keywords should be ",
+            absl::StrCat("Keyword '", GetName(token.GetLocationRange(), sql),
```

Comment:
Why you don't use `token.GetImage()` ? Here and in one more place. Then you could keep GetName for ASTNode only. (I wonder, if ASTNode also has a similar getter).

Reply:
I don't know why but I remember it was automaticly capitalized in tokenizer. Now I see thats not true in everycase. So changed it to `token.GetImage()`